python/osrd_schemas/osrd_schemas/infra.py

```diff
@@ -281,7 +285,8 @@ class SpeedSection(BaseObjectTrait):
     """
 
     speed_limit: Optional[float] = Field(
-        description="Speed limit (m/s) applied by default to trains that aren't in any specified category", gt=0
+        description="Speed limit (m/s) applied by default to trains that aren't in any specified category",
+        gt=0,
```

Comment:
This line wasn't longer than 120 characters as specified in osrd_schemas/pyproject.toml, it's weird that it's been reformated, as well as for the others

Reply:
I have fixed my formatter :partying_face: